src/handlers/guild.ts

```diff
@@ -622,3 +628,144 @@ export function getGuild(guildID: string, counts = true) {
     { with_counts: counts },
   ) as Promise<UpdateGuildPayload>;
 }
+
+/** Returns the guild template if it exists */
+export function getGuildTemplate(
+  guildID: string,
+  templateCode: string,
+) {
+  return RequestManager.get(
+    `${endpoints.GUILD_TEMPLATES(guildID)}/${templateCode}`,
+  ) as Promise<Template>;
+}
+
+/**
+ * Create a new guild based on a template
+ * NOTE: This endpoint can be used only by bots in less than 10 guilds.
+ */
+export async function createGuildFromTemplate(
+  templateCode: string,
+  data: CreateGuildFromTemplate,
+) {
+  if (cacheHandlers.size("guilds") >= 10) {
```

Comment:
```suggestion
  if (await cacheHandlers.size("guilds") >= 10) {
```

Reply:
await this in cases of custom cache someone stores it on redis and needs an await

Reply:
@Skillz4Killz, size() is not an async function.